template.yaml

```diff
@@ -50,6 +50,13 @@ objects:
           volumeMounts:
             - mountPath: /opt/janusgraph-0.2.0-hadoop2/db
               name: pv-db
+          resources:
+            limits:
+              cpu: '2'
```

Comment:
#moar!!

Reply:
Otherwise janusgraph was really sad and refused to start in reasonable time :(